pages/secret-manager/how-to/recover-secrets.mdx

```diff
@@ -0,0 +1,38 @@
+---
+meta:
+  title: How to recover secrets scheduled for deletion
+  description: Discover how to securely delete secrets using the Scaleway console. Follow these steps to manage your sensitive data effectively.
+content:
+  h1: How to recover secrets scheduled for deletion
+  paragraph: Discover how to securely delete secrets using Scaleway's intuitive console. Follow these steps to manage your sensitive data effectively.
+tags: secret sensitive-data scheduled-deletion recover-secret
+dates:
+  validation: 2025-03-27
+  posted: 2025-03-27
+categories:
+  - identity-and-access-management
+---
+
+This page shows you how to recover secrets scheduled for deletion using the Scaleway [console](https://console.scaleway.com). Once you schedule a secret for deletion, it enters a 7 day pending deletion period, during which you can still recover it.
+After this retention period, the secret and all its versions are permanently deleted.
+
+<Message type="important">
+  Scheduled deletion deletes ephemeral secrets and their versions immediately
+</Message>
+
+<Macro id="requirements" />
+
+- A Scaleway account logged into the [console](https://console.scaleway.com)
+- [Owner](/iam/concepts/#owner) status or [IAM permissions](/iam/concepts/#permission) allowing you to perform actions in the intended Organization
+- [Generated an API key](/iam/how-to/create-api-keys/) and enabled the `SecretManagerFullAccess` [permission set](/iam/reference-content/permission-sets/)
```

Comment:
optional - this prerequisite is only if you are doing it on API which is not the case for this tutorial